__tests__/data/mp_input.json

```diff
@@ -724,7 +726,9 @@
           "city": "Disney",
           "country": "USA",
           "email": "mickey@disney.com",
-          "firstname": "Mickey"
+          "first_name": "Mickey",
```

Comment:
Is this test-case to check if `firstName` is sent with wrong key ?

Reply:
In this test case we are checking support for `snake_case` aswell